src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php

```diff
@@ -120,45 +131,41 @@ public function buildForm(FormBuilderInterface $builder, array $options)
-                'label' => $this->trans('Preview', 'Admin.Actions'),
-                'icon' => 'remove_red_eye',
-                'attr' => [
-                    'class' => 'btn-outline-secondary preview-url-button',
-                    'data-seo-url' => $seoUrl,
-                    'disabled' => empty($productId),
-                ],
-            ])
             ->add('new_product', IconButtonType::class, [
-                'label' => $this->trans('New product on experimental page', 'Admin.Catalog.Feature'),
+                'label' => $this->trans('Add new product', 'Admin.Catalog.Feature'),
                 'type' => 'link',
                 'icon' => 'add_circle_outline',
                 'attr' => [
-                    'class' => 'btn-outline-secondary new-product',
+                    'class' => 'btn-outline-secondary new-product-button' . (empty($productId) ? ' disabled' : ''),
                     'href' => $this->router->generate('admin_products_v2_create'),
-                    'disabled' => empty($productId),
                 ],
             ])
-            ->add('standard_page', IconButtonType::class, [
-                'label' => $this->trans('Back to standard page', 'Admin.Catalog.Feature'),
+            ->add('cancel', IconButtonType::class, [
+                'label' => $this->trans('Cancel', 'Admin.Actions'),
+                'icon' => 'undo',
                 'type' => 'link',
                 'attr' => [
-                    'class' => 'btn-outline-secondary',
-                    'href' => $standardPageUrl,
+                    'href' => $page,
+                    'class' => 'btn-secondary cancel-button' . (empty($productId) ? ' disabled' : ''),
+                    'disabled' => true,
                 ],
-            ])
-            // These two inputs are displayed separately
-            ->add('active', SwitchType::class, [
-                'label' => false,
-                'choices' => [
-                    $this->trans('Offline', 'Admin.Global') => false,
-                    $this->trans('Online', 'Admin.Global') => true,
+            ])// These two inputs are displayed separately
+            ->add('preview', IconButtonType::class, [
+                'label' => $this->trans('Preview', 'Admin.Actions'),
+                'icon' => 'remove_red_eye',
+                'type' => 'link',
+                'attr' => [
+                    'target' => '_blank',
+                    'href' => $productPreviewUrl,
+                    'class' => 'btn-outline-secondary preview-url-button' . (empty($productId) ? ' disabled' : ''),
+                    'data-seo-url' => $seoUrl,
                 ],
             ])
             ->add('save', SubmitType::class, [
                 'label' => $options['active'] ? $this->trans('Save and publish', 'Admin.Actions') : $this->trans('Save', 'Admin.Actions'),
                 'attr' => [
                     'data-toggle' => 'pstooltip',
+                    'accesskey' => 's',
```

Comment:
Is this to handle the shortcut? I tried on my mac but I can't use it correctly Besides it makes me realize that the shortcuts don't seem to be handled the same way depending on the browser and OS so I guess the wording should adapted to the context
@MatShir what do you think?